src/index.ts

```diff
@@ -51,9 +39,12 @@ export {
   getGlobalScope,
   // getIsolationScope, TODO: Verify why it's not working
```

Comment:
The isolation scope is the most important in JS v8, can you share what's broken at the moment?

All the static apis `Sentry.addBreadcrumb` and other work on the isolation scope.

Reply:
I tried to capture a message and got this 
vendor.js:133798 Sentry Logger [warn]: No client configured on scope - will not capture message!

Reply:
I was testing getIsolationScope on a way that it wasn't designed so I am adding it back since it is working.